css-layout-classes/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1>Choose your character!</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="one-third1">
+        <img src="images/link.png" alt="Guy who everyone calls Zelda">
+      </div>
+      <div class="one">
+        <h2>Super Smash Bros. Ultimate</h2>
+        <p>Super Smash Bros Ultimate received universal acclaim from both critics and players, with some critics calling it the best installment in the series. It received praise for its large amount of content and fine-tuning of existing Smash gameplay elements, although its online mode was widely criticized.</p>
+      </div>
+      <div class="one-third1">
+        <img src="images/yoshi.png" alt="green dinosaur with shoes">
+      </div>
+    </div>
+    <div class="row">
+      <div class="four">
+        <img src="images/kirby.png" class="pics">
+        <img src="images/fox.png" class="pics">
+        <img src="images/pikachu.png" class="pics">
+        <img src="images/ness.png" class="pics">
+      </div>
+    </div>
+    <div class="row">
+      <div class="two-third">
```

Comment:
Think this should be `two-thirds`, I don't think you have a css class `two-third`.